net/zerotier/src/opnsense/mvc/app/views/OPNsense/Zerotier/index.volt

```diff
@@ -42,37 +49,55 @@ POSSIBILITY OF SUCH DAMAGE.
+            toggleNetworksTab(data['result']);
         });
 
-        $("#reconfigureZerotier").click(function() {
-            $("#reconfigureZerotierProgress").addClass("fa fa-spinner fa-pulse");
-            ajaxCall(url="/api/zerotier/zerotier/reconfigureZerotier", sendData={}, callback=function(data, status) {
-                ajaxCall(url="/api/zerotier/zerotier/status", sendData={}, callback=function(data,status) {
-                    updateServiceStatusUI(data['status']);
+        $("#save").click(function() {
+            $("#saveProgress").addClass("fa fa-spinner fa-pulse");
+            saveFormToEndpoint(url="/api/zerotier/zerotier/set", formid="global", callback_ok=function(data, status) {
+                ajaxCall(url="/api/zerotier/zerotier/status", sendData={}, callback=function(data, status) {
+                    updateServiceStatusUI(data['result']);
+                    toggleNetworksTab(data['result']);
                 });
-                $("#reconfigureZerotierProgress").removeClass("fa fa-spinner fa-pulse");
-                if (status != "success" || data['status'] != 'OK') {
-                    BootstrapDialog.show({
-                        type: BootstrapDialog.TYPE_WARNING,
-                        title: "{{ lang._('Error reconfiguring Zerotier') }}",
-                        message: data['status'],
-                        draggable: true
-                    });
-                }
+                $("#saveProgress").removeClass("fa fa-spinner fa-pulse");
             });
         });
+
+        function toggleNetworksTab(status) {
+            switch(status) {
+                case "disabled":
+                case "service_not_enabled":
+                    $('#ztNetworks').addClass("disabled");
+                    $('#ztNetworksLink').removeAttr("data-toggle");
+                    break;
+                default:
+                    $('#ztNetworks').removeClass("disabled");
+                    $('#ztNetworksLink').attr("data-toggle", "tab");
+            }
+        };
+
     });
 
 </script>
 
 <ul class="nav nav-tabs" data-tabs="tabs" id="maintabs">
-    <li class="active"><a data-toggle="tab" href="#networks">{{ lang._('Networks') }}</a></li>
+    <li id="ztGlobal" class="active"><a data-toggle="tab" href="#global">{{ lang._('Global') }}</a></li>
+    <li id="ztNetworks"><a id="ztNetworksLink" data-toggle="tab" href="#networks">{{ lang._('Networks') }}</a></li>
 </ul>
 
 <div class="tab-content content-box tab-content">
-    <div id="networks" class="tab-pane fade in active">
+    <div id="global" class="tab-pane fade in active">
+        <div class="content-box" style="padding-bottom: 1.5em;">
+            {{ partial("layout_partials/base_form", ['fields': globalForm, 'id': 'global']) }}
+            <hr/>
+            <div class="col-md-12">
+                <button class="btn btn-primary" id="save" type="button"><b>{{ lang._('Save') }}</b><i id="saveProgress" class=""></i></button>
```

Comment:
there is a space missing between "&lt;/b&gt;" and "&lt;i"

Reply:
Fixed. Will commit shortly.